### ISSUE TITLE
Is This Still a Work-In-Progress?

### ISSUE THREAD

**Issue by BillyMouser** [Salandora/OctoPrint-PanTilt#1]
I stumbled upon this plug-in, looking for the pan-tilt functionality with Octoprint, After installing this, i was forced to manually uninstall it as it broke the functionality of Octoprint's web gui. Is this something you are still working on?

**Comment by Salandora** [comment]
Yeah still a workin progress but actually no time to do somethong on it.
